system_test/aest_sync_SUITE.erl

```diff
@@ -625,18 +627,18 @@ net_split_mining_power(Cfg) ->
     SyncLength = 20,
     ExtraLength = 3,
 
-    Net1Nodes = [net1_node1, net1_node2],
-    Net2Nodes = [net2_node1, net2_node2, net2_node3, net2_node4],
+    Net1Nodes = [net1_node1],
+    Net2Nodes = [net2_node1, net2_node2, net2_node3],
     AllNodes = Net1Nodes ++ Net2Nodes,
 
-    setup_nodes([?NET1_NODE1, ?NET1_NODE2, ?NET2_NODE1,
-                 ?NET2_NODE2, ?NET2_NODE3, ?NET2_NODE4], Cfg),
```

Comment:
As now unused, may I suggest deleting NET2_NODE4 macro definition?

Reply:
No, cannot delete it, because it is refered to in the other nodes.

Reply:
I do not see net2_node4 referred to in the whole https://github.com/aeternity/epoch/blob/acc7fd0651a3e6ac79e5195126494519f25fbe6a/system_test/aest_sync_SUITE.erl